nn-git-log/README.md

```diff
@@ -0,0 +1,113 @@
+# Git Log
+
+Now that you know how to do the basics in git and started working on the repo, a few commits in and you might ask yourslef, how do we visualize these commits and changes we have made in the repository?  
+
+That is where this chapter, and the subsequent, `Git Diffs` come in. These commands give you a way to visualize the commits and changes made in your git respository.  
```

Comment:
> I believe that we don't have to create a new root chapter. We could probably move them into `07-advanced-git/` and maybe have them as separate folders instead? So something like `07-01-git-log/` and house all of the resources within that folder. I can edit the home `README` to include the subsections instead and have it link to each `README` separately.

I think this makes more sense in grouping these chapters together since its more on advanced git commands. Although I think they should still have their own `res` folder, this way we don't end up with all the images together in one place.  

> Alternatively, we could just house everything under the same document and have one giant `README` and instead store the resources in separate sub-folders under `res/` like `res/git-log/`.

In line with the reason above, it might also make development on additional chapters harder, since everything is in one big `README.md`.  

We could add a summary `README.md`  to give new readers a sense of what advanced chapters might be better for them. So, in the end, the file structure would look something like this.  
```
.
│
├── ...
├── 07-advanced-git
│   ├── 07-01-git-log
│   │   ├── res
│   │   │   └── ...
│   │   └── README.md
│   ├── 07-02-...
│   ├── 07-03-...
│   └── README.md
└── ...
```

Reply:
Yeap, I was referring to exactly that in the simple diagram you included :+1:

Reply:
Resolving this as the changes have been reflected :+1: